python/tvm/driver/tvmc/micro.py

```diff
@@ -236,18 +236,26 @@ def drive_micro(args):
     args.subcommand_handler(args)
 
 
+def get_project_dir(args: argparse.Namespace) -> str:
```

Comment:
That function will be used  in other contexts too, beyond `tvmc micro`. For instance, in `tvmc run ---device micro ...`, hence it should be moved to `common.py` and should not rely on any specific `Namespace` attribute, so it must accept a `pathlib.Path` or simply a `str` type.

Reply:
moved the function and changed input type.